src/components/App.jsx

```diff
@@ -12,20 +12,45 @@ import LoadingSpinner from './LoadingSpinner';
 
 import { PROJECT_STATUS } from '../ducks/project';
 import { WORKFLOW_STATUS } from '../ducks/workflow';
+import { SPLIT_STATUS } from '../ducks/splits';
+import GALogAdapter from '../lib/ga-log-adapter';
+import GoogleLogger from '../lib/GoogleLogger';
+import GeordiLogAdapter from '../lib/geordi-log-adapter';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.googleLogger = null;
+  }
+
   returnSomething(something) { // eslint-disable-line class-methods-use-this
     return something;
   }
 
+  getChildContext() {
+    return ({ googleLogger: this.googleLogger });
+  }
+
+  componentWillMount() {
```

Comment:
Is there any condition in which we need to worry about the Component un-mounting? Do we need to unsubscribe from the GoogleLogger, or is this extremely unlikely? (e.g. if App unmounts, it means the darn window has closed.)

Reply:
Ahh! Yes. I need to address this actually and I'm glad you brought it up. The `forget` function needs to be utilized in case of a user change. I will update this branch accordingly.